app/views/users/phone_setup/spam_protection.html.erb

```diff
@@ -58,3 +58,5 @@
        new_tab: true,
      ).with_content(t('two_factor_authentication.phone_verification.troubleshooting.learn_more')) %>
 <% end %>
+
+<%= render 'shared/cancel_or_back_to_options' %>
```

Comment:
We already have a "Choose another authentication method" in the troubleshooting options above.

Should we... ?

- Remove the troubleshooting option?
- Or, avoid showing the Cancel link in account creation?

Reply:
I think we can avoid showing the cancel link in account creation.